portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,16 @@ function nextImage() {
   imageContainer.src = currentImage;
   
 }
+
+/**
+ * Fetches data and adds to html
+ */
+function getHelloNameData() {
+  fetch('/data').then(response => response.json()).then((hello) => {
+      const msgContainer = document.getElementById('hello-container');
+
+      for (i = 0; i < hello.length; i++) {
+        msgContainer.innerHTML += hello[i];
```

Comment:
Should we add spacing between each hello here? 
Also consider using a for-each loop since you don't need the index for anything.